create_images.sh

```diff
@@ -15,35 +15,35 @@ echo "==> Generating images for patch level $_os_patch_level"
 
 # mkdir -p $(dirname $_boot_out)
 
-if [ "$_has_dtbo" = "true" ]; then
-    _dts_folder=$(realpath $_out/arch/arm64/boot/dts/qcom)
-    _files=$(find $_dts_folder -iname "*.dtbo")
+if [[ "$_has_dtbo" == "true" ]]; then
+    _dts_folder=$(realpath "$_out"/arch/arm64/boot/dts/qcom)
+    _files=$(find "$_dts_folder" -iname "*.dtbo")
     echo "==> Creating dtboimg from $_files"
-    python2 $_kernel_path/scripts/mkdtboimg.py create $_device-dtbo.img --page_size=$BOARD_KERNEL_PAGESIZE $_files
+    python2 "$_kernel_path"/scripts/mkdtboimg.py create "$_device-dtbo".img --page_size="$BOARD_KERNEL_PAGESIZE" "$_files"
```

Comment:
```suggestion
    python2 "$_kernel_path/scripts/mkdtboimg.py" create "$_device-dtbo.img" --page_size="$BOARD_KERNEL_PAGESIZE" "$_files"
```

(Note that `-dtbo` wasn't part of the expansion, this one should be read as `${_device}-dtbo.img`)

Perhaps we should see if this script can finally handle `python3` too and/or has a `/usr/bin/env` shebang.